player.js

```diff
@@ -171,7 +171,7 @@ export class CodeRadio {
 
   getNowPlaying() {
     // To prevent browser based caching, we add the date to the request, it won't impact the response
-    fetch(`/app/api/nowplaying?t=${new Date().valueOf()}`)
+    fetch(`https://github.com/slact/nchan.js/blob/master/NchanSubscriber.js`)
```

Comment:
I don't like the idea of depending on GitHub directly. They will simply blacklist our domain. Have this as a local or same domain dependency.

Reply:
Also I think you meant to replace the URL with this: `https://coderadio-admin.freecodecamp.org/api/nowplaying_static/coderadio.json`

Reply:
Finally you can remove the changes to the `_redirects` file that is proxying the API.